common/achievement/util.ts

```diff
@@ -19,8 +21,18 @@ export function getAchievementImageKey(imageKey: string) {
     return join(ACHIEVEMENT_IMAGE_DEFAULT_PATH, `${imageKey}`);
 }
 
-export function getAchievementImageURL(imageKey: string) {
-    return accessURLForKey(imageKey);
+export async function getAchievementImageURL(template: Achievement_template, state?: achievement_state, achievementContext?: Prisma.JsonValue) {
+    const templateIdsForCourseImage = [10];
+    const { id, image, achievedImage } = template;
+    const courseId = achievementContext?.['courseId'];
+    if (courseId && templateIdsForCourseImage.includes(id)) {
+        const { imageKey } = await prisma.course.findUnique({ where: { id: Number(courseId) }, select: { imageKey: true } });
+        return imageKey ? accessURLForKey(imageKey) : courseDefaultImage;
```

Comment:
Do we always want to return the default course image if there is none set for the course? Or shall we fallback to the default achievement image?

Reply:
We do not have a default achievement image so using the default course image seems reasonable.

Reply:
We do not have a default achievement image so using the default course image seems reasonable.